lib/game.rb

```diff
@@ -0,0 +1,43 @@
+class Game
```

Comment:
Well done for completing the challenge! Your code works! You got the core logic done and have full rspec coverage